ansible_collections/arista/avd/tests/unit/plugins/filter/test_range_expand.py

```diff
@@ -9,7 +9,36 @@
 
 from ansible_collections.arista.avd.plugins.filter.range_expand import AnsibleFilterError, FilterModule, range_expand
 
-RANGE_TO_EXPAND_INVALID_VALUES = [True, {"key": "value"}, 33]
+RANGE_TO_EXPAND_INVALID_VALUES = [
```

Comment:
Unrelated changes

Reply:
doh! (bis)

Reply:
Fixed in [733156c](https://github.com/aristanetworks/ansible-avd/pull/2907/commits/733156cdbc2ccc64a5a8145a7b42a6d989994c92)